src/tests/ftest/container/autotest.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from apricot import TestWithServers
+
+
+class ContainerAutotestTest(TestWithServers):
+    # pylint: disable=too-few-public-methods
+    """Tests container autotest.
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize object."""
+        super().__init__(*args, **kwargs)
+        self.pool = None
+        self.container = None
+        self.daos_cmd = None
```

Comment:
You don't really need these variables. You can just make them local.

Reply:
Ok